migrations/lib/converters/discourse/steps/tags.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Migrations::Converters::Discourse
+  class Tags < ::Migrations::Converters::Base::ProgressStep
+    attr_accessor :source_db
+
+    def max_progress
+      @source_db.count <<~SQL
+        SELECT COUNT(*) FROM tags
+      SQL
+    end
+
+    def items
+      @source_db.query <<~SQL
+        SELECT * FROM tags
+      SQL
+    end
+
+    def process_item(item)
+      IntermediateDB::Tag.create(
+        original_id: item[:id],
+        created_at: item[:created_at],
+        description: item[:description],
+        name: item[:name],
+        target_tag_id: item[:target_tag_id],
```

Comment:
I wonder if we should deviate from the Discourse DB here? `target_tag_id` is such a hard-to-remember name. I would have needed to look at the source to figure out that it's for synonyms.

Maybe rename the column (which we can't do yet; renaming is only implemented globally), or even add a `IntermediateDB::TagSynonym` module manually? I feel like this is one of the cases where we need to make it easier for devs working on converters to figure things out.

What do you think?